src/api/node/sse/types.rs

```diff
@@ -0,0 +1,63 @@
+use eventsource_stream::{Event, EventStreamError};
+use futures::stream::BoxStream;
+use serde::{Deserialize, Serialize};
+use tokio::sync::oneshot;
+
+//copied from casper-sidecar
+
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Hash)]
+pub enum EventType {
+    ApiVersion,
+    SidecarVersion,
+    BlockAdded,
+    TransactionAccepted,
+    TransactionProcessed,
+    TransactionExpired,
+    Fault,
+    FinalitySignature,
+    Step,
+    Shutdown,
+}
+
+/// Casper does not expose SSE types directly, so we have to reimplement them.
+/// Source: https://github.com/casper-network/casper-node/blob/8a9a864212b7c20fc17e1d0106b02c813ffded9d/node/src/components/event_stream_server/sse_server.rs#L56.
```

Comment:
Consider just using `casper-sidecar` instead of doing this: https://github.com/casper-network/casper-sidecar/tree/feat-2.0